tests/integration/integration.go

```diff
@@ -120,6 +122,11 @@ func setInstances() {
 	subnetID := ids.GenerateTestID()
 	chainID := ids.GenerateTestID()
 
+	vmConfig := vm.NewConfig()
+	cb, err := json.Marshal(vmConfig)
+	require.NoError(err)
+	configBytes = cb
```

Comment:
Can we remove this and revert this part of the diff since this is back to relying on the defaults?